src/WCSimEventAction.cc

```diff
@@ -1340,29 +1345,29 @@ void WCSimEventAction::FillRootEvent(G4int event_id,
           if (ttime > WCTM->GetTriggerTime(1) + 950. && WCTM->GetTriggerTime(2) + 950. > ttime) choose_event = 2;
           if (choose_event >= ngates) choose_event = ngates - 1; // do not overflow the number of events
 
-      }
-
-      wcsimrootevent = wcsimrootsuperevent->GetTrigger(choose_event);
-      wcsimrootevent->AddTrack(ipnu,
-                               flag,
-                               mass,
-                               mommag,
-                               energy,
-                               startvol,
-                               stopvol,
-                               dir,
-                               pdir,
-                               stop,
-                               start,
-                               parentType,
-                               ttime,
-                               id,
-                               idPrnt,
-                               trj->GetBoundaryPoints(),
-                               trj->GetBoundaryKEs(),
-                               trj->GetBoundaryTimes(),
-                               trj->GetBoundaryTypesAsInt());
-
+          }
```

Comment:
beware of the indentations (and in all other places)

Reply:
👍